app/containers/InAppNotification/index.js

```diff
@@ -14,7 +14,7 @@ const InAppNotification = memo(() => {
 		const state = Navigation.navigationRef.current?.getRootState();
 		const route = getActiveRoute(state);
 		if (payload.rid) {
-			if (route?.name === 'RoomView' && route.params?.rid === payload.rid) {
+			if ((route?.name === 'RoomView' || route?.name === 'JitsiMeetView') && route.params?.rid === payload.rid) {
```

Comment:
You should test if it's on call on a separate condition.
The way you did, it wouldn't work when an in-app notification is received from another room.

Reply:
Fixed it.